script.js

```diff
@@ -1,11 +1,131 @@
+    images: ['img/Rectangle 21.png', 'img/22.png', 'img/23.png', 'img/24.png'],
+    technology: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linkToLiveVersion: '#',
+    linkToSource: 'https://github.com/Kashif-Saleem-Ghuman',
+  },
+  {
+    projectTitle: 'Project #5',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum molestias, quod recusandae asperiores similique quos aperiam aspernatur, ipsam autem fuga minima libero dolor accusantium corporis possimus quaerat repellat harum distinctio! Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum molestias, quod recusandae asperiores similique quos aperiam aspernatur, ipsam autem fuga minima libero dolor accusantium corporis possimus quaerat repellat harum distinctio!',
+    mainImage: './img/25.png',
+    images: ['img/24.png', 'img/23.png', 'img/22.png', 'img/Rectangle 21.png'],
+    technology: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linkToLiveVersion: 'https://kashif-saleem-ghuman.github.io/My-Portfolio',
+    linkToSource: 'https://github.com/Kashif-Saleem-Ghuman',
+  },
+
+  {
+    projectTitle: 'Project #6',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum molestias, quod recusandae asperiores similique quos aperiam aspernatur, ipsam autem fuga minima libero dolor accusantium corporis possimus quaerat repellat harum distinctio! Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum molestias, quod recusandae asperiores similique quos aperiam aspernatur, ipsam autem fuga minima libero dolor accusantium corporis possimus quaerat repellat harum distinctio!',
+    mainImage: './img/26.png',
+    images: ['img/Rectangle 21.png', 'img/22.png', 'img/23.png', 'img/24.png'],
+    technology: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+    linkToLiveVersion: '#',
+    linkToSource: 'https://www.javascripttutorial.net/javascript-objects/',
+  }];
+
+const modal = document.querySelector('#modal');
+
+function openModal(num = null) {
+  if (num != null) {
+    const { technology } = projectCard[num];
+    let technologyList = '';
+    technology.forEach((listedItem) => {
+      technologyList += `<li>${listedItem}</li>`;
+    });
+
+    const { images } = projectCard[num];
+    let imagesList = '';
+    images.forEach((listedImage) => {
+      imagesList += `<img src="${listedImage}"></img>`;
+    });
+
+    document.getElementById('modal-project-name').innerHTML = projectCard[num].projectTitle;
+    document.getElementById('modal-languages').innerHTML = technologyList;
+    document.getElementById('Picture').src = projectCard[num].mainImage;
+    document.querySelector('.modal-image-preview').innerHTML = imagesList;
+    document.getElementById('project-live').href = projectCard[num].linkToLiveVersion;
+    document.getElementById('modal-project-source').href = projectCard[num].linkToSource;
+
+    modal.classList.add('active');
+  }
+}
+
+function closeModal(index = null) {
```

Comment:
[OPTIONA] Kindly use ES6 arrow function to show that you are updated as it would help you in the industry and even in React module.